designcompose/src/main/java/com/android/designcompose/squoosh/SquooshRoot.kt

```diff
@@ -259,6 +262,11 @@ fun SquooshRoot(
     val rootLayoutId = remember(docId) { SquooshLayout.getNextLayoutId() * 100000000 }
     val layoutCache = remember(docId) { HashMap<Int, Int>() }
     val layoutValueCache = remember(docId) { HashMap<Int, Layout>() }
+    val keyEventTracker = remember(docId) { KeyEventTracker() }
```

Comment:
Do we get in trouble if we have multiple `DesignRoot`s showing components from the same doc id?

Reply:
Probably -- I could add the `rootNodeQuery` to the `remember`.